emacs/go-autocomplete.el

```diff
@@ -202,6 +207,9 @@
     (requires . 0)
     (cache)))
 
+(if (string= "" ac-go-gocode-bin)
+    (setq ac-go-gocode-bin "gocode"))
+
```

Comment:
Second argument of `defcustom` is the default value (which you decided to set to `t`). You don't need that if here.